platform/00-kubernetes-minikube-setup.sh

```diff
@@ -16,11 +16,39 @@ HELM_FILE_NAME=helm-v3.7.1-linux-amd64.tar.gz
 HELM_URL=https://get.helm.sh/$HELM_FILE_NAME
 HELM_SHA256=6cd6cad4b97e10c33c978ff3ac97bb42b68f79766f1d2284cfd62ec04cd177f4
 
-TKN_VERSION=0.21.0
+TKN_VERSION=v0.21.0
 TKN_FILE_NAME=tkn_0.21.0_Linux_x86_64.tar.gz
-TKN_URL=https://github.com/tektoncd/cli/releases/download/v$TKN_VERSION/$TKN_FILE_NAME
+TKN_URL=https://github.com/tektoncd/cli/releases/download/$TKN_VERSION/$TKN_FILE_NAME
 TKN_SHA256=2158a202e4b04ff73e6427b565355c7bfc8cbe16dc7058a0414fb16e7b97008c
 
+KUBECTL_VERSION=v1.22.3
+KUBECTL_FILE_NAME=kubectl
+KUBECTL_URL=https://dl.k8s.io/release/$KUBECTL_VERSION/bin/linux/amd64/kubectl
+KUBECTL_VALIDATE_CHECKSUM_URL=$KUBECTL_URL.sha256
+
+case $(uname -m) in
+  x86_64)
+    COSIGN_ARCH=amd64
+  ;;
+  arm)
+    COSIGN_ARCH=arm
+  ;;
+  aarch64)
```

Comment:
I think this may need to be `arm64`. At least it does for my M1-Pro MacBook.

Reply:
Hey Brad, that arch switch is only used for the Linux install and I checked the uname architecture responses, didn't see arm64. That said, in the above installs we don't support arm arch on Linux anyway.

Reply:
Fair enough. I do get the following on my MacBook Pro (M1-Pro):
```
$ uname -m
arm64
```